components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```
@@ -456,5 +453,27 @@ def main(argv):
                 return -1
 
 
+def _add_and_cache_dataset(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)